### PR TITLE
[Merged by Bors] - Update mainnet altair types test

### PR DESCRIPTION
**Pull request by divagant-martian**
## Issue Addressed
e895074ba updated the altair fork and now that we are a week away this test no longer panics.

## Proposed Changes
Remove the expected panic and explanatory note.

### CODE REVIEW

**Review by pawanjay176 — APPROVED** [review]

**Comment by divagant-martian** [comment]
The first workflow failed with a transport error starting the beacon node in windows in release `An attempt was made to access a socket in a way forbidden by its access permission`. Running again because it seems a random failure

**Review by paulhauner — APPROVED** [review]
Great, thanks Diva!

In my comment I'd said that the test would fail until Lighthouse had set the Altair fork epoch. Whilst that was true to some degree, I didn't consider that the test would also continue to fail until web3signer had also added Altair support.

Web3signer released [2.10.0](https://github.com/ConsenSys/web3signer/releases/tag/21.10.0) ~13hrs ago which provides "support for Altair fork in mainnet". So, CI would have pulled this release, ran the tests and then violated the `should_panic`.

TL;DR, I think this PR is good and I understand why that test started failing (or passing? :shrug:) all of a sudden.

bors r+
